src/main.rs

```diff
@@ -4,6 +4,6 @@ mod config;
 mod constants;
 mod utils;
 
-pub fn main() {
+pub fn main() -> ! {
```

Comment:
Wait, is `!` stable yet?

Reply:
My understanding is that it is not yet stable as a type, but it is stable when used to indicate that a function never returns, like how I am using it here